src/gui/editorwidgets/qgsvaluemapconfigdlg.cpp

```diff
@@ -161,12 +173,12 @@ void QgsValueMapConfigDlg::updateMap( const QMap<QString, QVariant> &map, bool i
     ++row;
   }
 
-  for ( QMap<QString, QVariant>::const_iterator mit = map.begin(); mit != map.end(); ++mit, ++row )
+  for ( const auto &pair : list )
```

Comment:
I know it's perfectly pointless in this case but someone seemed to prefer `qgis::as_const` for readability in the past, is there a reason for not using it here?